content/docs/quickstart/_index.md

```diff
@@ -109,7 +109,7 @@ Please make sure you have `kubectl` installed (see [kubernetes.io/docs/tasks/too
     
     or if you are on Windows
     ```console
-    ./keptn version
```

Comment:
Same as above, the `./` (or `.\`) is there for a reason on Windows

Reply:
see above.